conf/general-example.ubuntu

```diff
@@ -20,6 +20,9 @@ define ("COOKIEDOMAIN", "dev.openaustralia.org");
 
 define ("CONTACTEMAIL", "contact@openaustralia.org");
 
+// File system path to the html email tamplate
+define ("HTML_EMAIL_TEMPLATE","/srv/www/openaustralia/twfy/www/includes/easyparliament/templates/emails/html_email_template.html");
```

Comment:
I didn't feel good having filenames hardcoded in code, but in fact that's
happening elsewhere anyway.
I guess it could have leveraged the INCLUDE path.

On Sun, Feb 15, 2015 at 3:06 PM, Henare Degan <notifications@github.com>
wrote:

> In conf/general-example.ubuntu
> <https://github.com/openaustralia/twfy/pull/11#discussion_r24719523>:
>
> > @@ -20,6 +20,9 @@ define ("COOKIEDOMAIN", "dev.openaustralia.org");
> >
> >  define ("CONTACTEMAIL", "contact@openaustralia.org");
> >
> > +// File system path to the html email tamplate
> > +define ("HTML_EMAIL_TEMPLATE","/srv/www/openaustralia/twfy/www/includes/easyparliament/templates/emails/html_email_template.html");
>
> Since the templates are always in the code repository at the same
> location, I don't think this needs to be configurable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openaustralia/twfy/pull/11/files#r24719523>.
>



-- 
+61 468 335597
*LinkedIn Profile : Matt Joyce
<http://au.linkedin.com/pub/matthew-joyce/6/4b5/6a4/>*